app/views/dashboard/index.html.erb

```diff
@@ -97,9 +97,9 @@
 <% end %>
 
 <% if @is_spo %>
-  <h2 class="govuk-heading-m no-bottom-margin">Manage staff</h2>
-  <hr class="govuk-section-break govuk-section-break--visible">
-  <div class="govuk-grid-row dashboard-row">
+  <h2 class="govuk-heading-m no-bottom-margin govuk-!-margin-top-3">Manage staff</h2>
+  <hr class="govuk-section-break govuk-section-break--visible govuk-!-margin-bottom-5">
+  <div class="govuk-grid-row card-groups">
```

Comment:
You've used `card-groups` here rather than `card-group`. Is this a typo?

```suggestion
  <div class="govuk-grid-row card-group">
```

Reply:
@ollietreend Oh well spotted. It is a typo!